xgbutil error not handled => crash

`time="2023-04-06T07:29:26-04:00" level=warning msg="GetProperty: No such property 'WM_CLASS' on window ca00003."
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0x5868df]

goroutine 1 [running]:
main.shouldIgnore(0xc00ca00003, 0xc006c00000)
	/home/mdg/src/zentile/client.go:154 +0xdf
main.(*tracker).populateClients(0xc0000574f0)
	/home/mdg/src/zentile/tracker.go:32 +0x98
main.(*tracker).handleClientUpdates(0xc0000574f0, 0xc00015c140, 0xc0000ee440)
	/home/mdg/src/zentile/tracker.go:92 +0x86
github.com/BurntSushi/xgbutil/xevent.PropertyNotifyFun.Run(0xc000057500, 0xc00015c140, 0x5c3560, 0xc0000ee440)
`

here's a full stack trace

```WARN[39363] GetProperty: No such property 'WM_CLASS' on window da00002. 
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0x552b16]

goroutine 1 [running]:
main.shouldIgnore(0x20a080?)
	/home/mdg/src/zentile/client.go:154 +0xb6
main.(*tracker).populateClients(0xc00020a080)
	/home/mdg/src/zentile/tracker.go:32 +0xae
main.(*tracker).handleClientUpdates(0xc00020a080, 0xc00005ee48?, {0x40e7c8?})
	/home/mdg/src/zentile/tracker.go:92 +0x5e
github.com/BurntSushi/xgbutil/xevent.PropertyNotifyFun.Run(0x578e20?, 0xc000206240?, {0x58b620?, 0xc000292ea0?})
	/home/mdg/go/pkg/mod/github.com/!burnt!sushi/xgbutil@v0.0.0-20160919175755-f7c97cef3b4e/xevent/callback.go:311 +0x38
github.com/BurntSushi/xgbutil/xevent.runCallbacks(0xc00015a140, {0x58b620, 0xc000292ea0}, 0xc00015a140?, 0x5e8ca8?)
	/home/mdg/go/pkg/mod/github.com/!burnt!sushi/xgbutil@v0.0.0-20160919175755-f7c97cef3b4e/xevent/xevent.go:203 +0xb7
github.com/BurntSushi/xgbutil/xevent.processEventQueue(0xc00015a140, 0x0, 0x0)
	/home/mdg/go/pkg/mod/github.com/!burnt!sushi/xgbutil@v0.0.0-20160919175755-f7c97cef3b4e/xevent/eventloop.go:253 +0xb8f
github.com/BurntSushi/xgbutil/xevent.mainEventLoop(0xc00015a140, 0x0?, 0x6e1188?, 0x0)
	/home/mdg/go/pkg/mod/github.com/!burnt!sushi/xgbutil@v0.0.0-20160919175755-f7c97cef3b4e/xevent/eventloop.go:108 +0x48
github.com/BurntSushi/xgbutil/xevent.Main(...)
	/home/mdg/go/pkg/mod/github.com/!burnt!sushi/xgbutil@v0.0.0-20160919175755-f7c97cef3b4e/xevent/eventloop.go:52
main.main()
	/home/mdg/src/zentile/main.go:19 +0x45```

--- a/client.go
+++ b/client.go
@@ -116,7 +116,7 @@ func (c Client) Restore() {
 	c.MoveResize(geom.X(), geom.Y(), geom.Width(), geom.Height())
 }
 
-//  Activate makes the client the currently active window
+// Activate makes the client the currently active window
 func (c Client) Activate() {
 	ewmh.ActiveWindowReq(state.X, c.window.Id)
 }
@@ -148,6 +148,7 @@ func shouldIgnore(w xproto.Window) bool {
 	c, err := icccm.WmClassGet(state.X, w)
 	if err != nil {
 		log.Warn(err)
+		return true // c is nil and can't be used below
 	}
 
 	for _, s := range Config.WindowsToIgnore {

